skyportal/handlers/api/observation_plan.py

```diff
@@ -904,6 +904,81 @@ def get(self, observation_plan_request_id):
             return self.error('Observation plan not yet available.')
 
 
+class ObservationPlanFieldsHandler(BaseHandler):
+    @auth_or_token
+    def delete(self, observation_plan_request_id):
+        """
+        ---
+        description: Delete selected fields from the observation plan.
+        tags:
+          - observation_plan_requests
+        parameters:
+          - in: path
+            name: observation_plan_id
+            required: true
+            schema:
+              type: string
+          - in: query
+            name: fieldIds
+            nullable: false
+            schema:
+              type: list
+            description: |
+              List of field IDs to remove from the plan
+        responses:
+          200:
+            content:
+              application/json:
+                schema: SingleObservationPlanRequest
+        """
+
+        data = self.get_json()
+        field_ids_to_remove = data.pop('fieldIds', None)
```

Comment:
This should probably be a `get_query_argument` as the docstring suggests this is a query argument (which sounds right).